client/my-sites/patterns/components/category-not-found/index.tsx

```diff
@@ -8,16 +8,27 @@ import './style.scss';
 
 export const PatternsCategoryNotFound = () => {
 	const isLoggedIn = useSelector( isUserLoggedIn );
-	const translate_not_yet = useTranslate();
-
-	const comment =
-		'Message displayed when an invalid category was specified while searching block patterns';
+	const translate = useTranslate();
 
+	const emptyContentStrings = {
+		title: translate( "Oops! We can't find this category!", {
+			comment:
+				'Message displayed when an invalid category was specified while searching block patterns',
+		} ),
+		line: translate( "The category you are looking for doesn't exist.", {
+			comment:
+				'Message displayed when an invalid category was specified while searching block patterns',
+		} ),
+		action: translate( 'Browse all patterns', {
+			comment:
+				'Message displayed when an invalid category was specified while searching block patterns',
+		} ),
+	};
 	return (
 		<EmptyContent
-			title={ translate_not_yet( "Oops! We can't find this category!", { comment } ) }
-			line={ translate_not_yet( "The category you are looking for doesn't exist.", { comment } ) }
-			action={ translate_not_yet( 'Browse all patterns', { comment } ) }
```

Comment:
the `comment` variable here was triggering a `wpcalypso/i18n-no-variables` warning, so I've replaced it with the actual string. The component props felt really messy with that repetition, so i've moved it (and the actual translation) into a separate variable.

I don't love that we have to repeat the same string over and over for the `comment`, but I don't see another way around that warning 🤔